react-frontend/src/components/Nav/navbar.js

```diff
@@ -42,6 +45,10 @@ function NavBar() {
     }
   };
 
+  let logoPath = isMobile
+    ? require('../../images/logo-banner.png')
```

Comment:
hmm i'd prefer an import for each image at the top of the component rather than a dyanmic import. this way it only has to run those functions once. Here it will run it every time the header is rendered